packages/wallet/backend/src/account/service.ts

```diff
@@ -70,18 +70,25 @@ export class AccountService implements IAccountService {
       throw new NotFound()
     }
 
-    const result = await this.gateHubClient.createWallet(
-      user.gateHubUserId,
-      args.name
-    )
+    const result = await this.gateHubClient.getWalletForUser(user.gateHubUserId)
+    // The user should not have more than one wallet address anyway
+    // Only on production GateHub wallet address would already exist
+    let gateHubWalletId = result.wallets[0].address
+    if (!gateHubWalletId) {
```

Comment:
discussed to be changed to check if execution of function is for default card-holding account in production, where users are expected to have the gatehub wallet account already, instead of checking if wallet already exists 

as this would disrupt flow for users just creating new accounts